src/sc3_mpienv.c

```diff
@@ -423,3 +423,53 @@ sc3_mpienv_get_node_frank (const sc3_mpienv_t * m, int *node_frank)
   *node_frank = m->node_frank;
   return NULL;
 }
+
+sc3_error_t        *
+sc3_mpienv_get_node_num (const sc3_mpienv_t * m, int *node_num)
+{
+  SC3E_RETVAL (node_num, -1);
+  SC3A_IS (sc3_mpienv_is_setup, m);
+
+  *node_num = m->node_num;
+  return NULL;
+}
+
+sc3_error_t        *
+sc3_mpienv_get_num_nodes (const sc3_mpienv_t *m, int *num_nodes)
+{
+  SC3E_RETVAL (num_nodes, -1);
+  SC3A_IS (sc3_mpienv_is_setup, m);
+
+  *num_nodes = m->num_nodes;
+  return NULL;
+}
+
+sc3_error_t        *
+sc3_mpienv_get_node_sizes (const sc3_mpienv_t * m, int **node_sizes)
```

Comment:
Thanks! What would be your opininon about trying the `const int **` type?

Reply:
It makes sense to add it for safety reasons.